src/common-components/data/reducers.js

```diff
@@ -23,15 +23,21 @@ const reducer = (state = defaultState, action) => {
         ...state,
         thirdPartyAuthApiStatus: PENDING_STATE,
       };
-    case THIRD_PARTY_AUTH_CONTEXT.SUCCESS:
+    case THIRD_PARTY_AUTH_CONTEXT.SUCCESS: {
+      const extendedProfile = action.payload.optionalFields.extended_profile;
+      const extendedProfileArray = extendedProfile && JSON.parse(extendedProfile.replaceAll('\'', '"'));
```

Comment:
Looks good to me, but one question why we replacing `\` to `"`?

Reply:
The MFE context response we are getting is in `extended_profile="['city', 'state']"` format, and JSON parse cannot parse `'` single quotation marks, therefore it has to be converted to `"` double quotation marks before parsing.